server/src/testFixtures/java/io/crate/testing/SqlExpressions.java

```diff
@@ -72,11 +74,19 @@ public SqlExpressions(Map<RelationName, AnalyzedRelation> sources,
 
     public SqlExpressions(Map<RelationName, AnalyzedRelation> sources,
                           @Nullable FieldResolver fieldResolver,
-                          User user,
+                          User sessionUser,
                           AbstractModule... additionalModules) {
-        this.nodeCtx = createNodeContext(user, additionalModules);
+        this(sources, fieldResolver, sessionUser, List.of(), additionalModules);
+    }
+
+    public SqlExpressions(Map<RelationName, AnalyzedRelation> sources,
+                          @Nullable FieldResolver fieldResolver,
+                          User sessionUser,
+                          List<User> users,
```

Comment:
Think this would clarify the usage better:
```suggestion
                          List<User> additionalUsers,
```